.github/workflows/inspektor-gadget.yml

```diff
@@ -1719,8 +1719,14 @@ jobs:
         sudo apt-get install -y qemu-system-x86
         sudo chmod 666 /dev/kvm
     - name: Install vimto
+      # Instead of lmb.io/vimto@latest
+      # Use github.com/alban/vimto@alban_devpts
+      # Because of https://github.com/lmb/vimto/pull/26
```

Comment:
https://github.com/lmb/vimto/pull/26 has been merged, so we can remove the use of my fork.